smallrye-reactive-messaging-kafka/src/test/java/io/smallrye/reactive/messaging/kafka/base/MultipleInstance.java

```diff
@@ -0,0 +1,76 @@
+package io.smallrye.reactive.messaging.kafka.base;
+
+import java.lang.annotation.Annotation;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+import javax.enterprise.inject.Instance;
+import javax.enterprise.util.TypeLiteral;
+
+import org.jetbrains.annotations.NotNull;
```

Comment:
Do we really need this annotation?

Reply:
We have one in SmallRye if needed: https://github.com/smallrye/smallrye-common/tree/main/constraint/src/main/java/io/smallrye/common/constraint

Reply:
I removed it, it was added automatically by the IDE